test/dune

```diff
@@ -0,0 +1,4 @@
+(test
+  (name test)
+  (modules test)
```

Comment:
Is the `modules` stanza necessary here?

Reply:
As you have only one executable it's not. But if you ever want to add a second executable, then, you'll have to specify it explicitly. I can remove it if you want, but having it it's causing no harm. :)